docs/configuration.md

```diff
@@ -0,0 +1,63 @@
+## Configuration
+
+The btp-manager internal settings can be configured via CLI arguments or `ConfigMap`.
```

Comment:
I'm afraid, `via` is one of the forbidden words according to our content guidelines (https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology).